lectures/_config.yml

```diff
@@ -13,4 +13,3 @@ sphinx:
     html_theme: quantecon_book_theme
     html_static_path: ['_static']
     html_theme_options:
```

Comment:
hey @DrDrij we need to remove `html_theme_options` if there is nothing else nested below it.